docs/redoc/master/openapi.json

```diff
@@ -9411,7 +9411,8 @@
       "OperationDurationStatistics": {
         "type": "object",
         "required": [
-          "count"
+          "count",
+          "total_duration_micros"
```

Comment:
It's output-only structure and this field is never skipped, so the new schema is more accurate.